tst/components/ui/Chat/ChatBubble/index.test.tsx

```diff
@@ -42,6 +42,22 @@ describe('ChatBubble', () => {
     expect(name).not.toBeInTheDocument();
   });
 
+  it('should render timestamp in the ChatBubble component', () => {
+    const component = <ChatBubble content='test message' senderName='test user' timestamp='1:30 PM' variant='outgoing' />;
+    const { getByText } = renderWithTheme(lightTheme, component);
+    const timestamp = getByText('1:30 PM');
+
+    expect(timestamp).toBeInTheDocument();
+  });
+
+  it('should not render timestamp in the ChatBubble component if showName is false', () => {
```

Comment:
I'm not sure I the relationship between `showName` and showing the `timestamp`. I think you should be able to see a timestamp even if in some cases you don't show the name.